lib.py

```diff
@@ -36,6 +37,21 @@ def get_inventory():
     if not inventory_cmd:
         raise Exception("Failed to identify path of ansible-inventory")
 
+    inventory = None
+
+    # read and invalidate the inventory cache file
+    cache_file = os.environ.get("BASTION_ANSIBLE_INV_CACHE_FILE")
+    if cache_file:
+        cache = get_inventory_from_cache(
+            cache_file=cache_file,
+            cache_timeout=os.environ.get("BASTION_ANSIBLE_INV_CACHE_TIMEOUT", 60),
```

Comment:
```suggestion
            cache_timeout=int(os.environ.get("BASTION_ANSIBLE_INV_CACHE_TIMEOUT", 60)),
```

Else, you'll have a "TypeError: unsupported operand type(s) for -: 'int' and 'str'" when you set a custom BASTION_ANSIBLE_INV_CACHE_TIMEOUT

Reply:
I've applied your suggestion. Thanks!